### PR TITLE
Created a read me file

### PR DESCRIPTION
**Pull request by mariadurrani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+I'm passionate about creating beautiful and intuitive user experiences through web development. Whether it's crafting elegant designs or diving into the intricacies of code, I love every aspect of frontend development! 💻
+
+What You'll Find Here 📂   
+Projects: Explore my latest creations, from simple landing pages to interactive web applications.   
+Skills: Check out the technologies I'm proficient in, including HTML, CSS, JavaScript, and more!   
+About Me: Learn more about my background, interests, and aspirations in the world of frontend development.
+
+Let's Connect! 🤝
+
+I'm always eager to collaborate and learn from fellow developers! Feel free to reach out to me via:
```
Comment (Evgenios95) [inline README.md]: Heyyy nice job. I think it makes sense to also mention the list of technologies that we are working with in this project. Also some patterns that we use, maybe some architecture as well :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Welcome to Maria Durrani's Portfolio! 🚀
+
+Hey there! 👋 Welcome to my portfolio, where I showcase my journey as a budding frontend developer! 🌟
+
+About Me ℹ️  
+I'm passionate about creating beautiful and intuitive user experiences through web development. Whether it's crafting elegant designs or diving into the intricacies of code, I love every aspect of frontend development! 💻
+
+What You'll Find Here 📂   
+Projects: Explore my latest creations, from simple landing pages to interactive web applications.   
+Skills: Check out the technologies I'm proficient in, including HTML, CSS, JavaScript, and more!
```
Comment (Evgenios95) [inline README.md]: Maybe change the "proficient" to something else, as it needs tones of time to be proficient in those here :) I am not after 5 years of coding

Comment (mariadurrani) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (mariadurrani) [reply README.md]: Done

**Review by fuzzypawzz — APPROVED** [review]

**Pull request merged**